sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -56,6 +56,22 @@ void gnrc_sixlowpan_nd_init(gnrc_ipv6_netif_t *iface)
     gnrc_ndp_internal_send_rtr_sol(iface->pid, NULL);
 }
 
+void gnrc_sixlowpan_nd_router_set_rtr_adv(gnrc_ipv6_netif_t *netif, bool enable)
+{
+    ipv6_addr_t all_routers = IPV6_ADDR_ALL_ROUTERS_LINK_LOCAL;
```

Comment:
Me neither, that's exactly the point of local variables: they are discard-able, they only need to exist when they are needed. Global addresses on the other hand are always in memory, even if they are not in use.